### ISSUE TITLE
[mdc-tooltip] z-index

### ISSUE THREAD

**Issue by simonziegler** [material-components/material-components-web#6348]
## Bug report
Experiments with mdc-toolip fail to show tooltips until I set the z-index to a sufficiently high number, e.g. `style="z-index: 1000;"`.

### Your Environment:

| Software         | Version(s) |
| ---------------- | ---------- |
| MDC Web          | `"@material/tooltip": "8.0.0-canary.2e5711e04.0",`, otherwise everything is 7.0.0 |
| Browser          | Edgium |
| Operating System | Windows 10 |

### Additional context
Using `"@material/tooltip": "8.0.0-canary.2e5711e04.0",`.

### Possible solution
Consider z-index in the mdc-tooltip class.

**Comment by asyncLiz** [comment]
Do you have a reproduction of the problem? The tooltip works in Edge with this demo: https://jsbin.com/xinefixoqu/edit?html,output

Keep in mind that the tooltip [has a z-index already](https://github.com/material-components/material-components-web/blob/504af0ca835665db676d42ee8ad59a5dc94f359e/packages/mdc-tooltip/_tooltip.scss#L36). My suspicion is that there's something in your application that has an equal or higher z-index that is covering the tooltip.

**Comment by simonziegler** [comment]
Hi @asyncLiz, I do at least sort of at https://blazormdc.com/miscellaneous. Blazor MDC is a package bringing Material Theme to the Microsoft ASP.NET Blazor platform. The page that I reference is a demo page and you'll find that the icons near the top of the page have tooltips. The tooltips themselves are just under the `<body>` tag at the top of the '<app>` tag. I've set the z-index to 1000 manually, and guess what: now I've published the site if disable the z-index in developer tools the tooltip still shows correctly. 

🤦‍♂️

So I need to go back and remove the z-index and probably find that there's no issue after all! I'll close this issue now and if I find that there is actually a problem, I'll reopen in the future.

Thanks for the attention, Simon
